flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BucketWriter.java

```diff
@@ -55,7 +55,8 @@
   /**
    * This lock ensures that only one thread can open a file at a time.
    */
-  private static final Integer staticLock = new Integer(1);
+  private static final Integer STATIC_LOCK = new Integer(1);
+  @SuppressWarnings("squid:S1068")
   private Method isClosedMethod = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "isClosedMethod" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)